src/views/CampusSafety/views/LostAndFound/views/MissingItemEdit/index.tsx

```diff
@@ -119,6 +124,23 @@ const MissingItemFormEdit = () => {
     setShowConfirm(true);
   };
 
+  const handlePickup = async () => {
+    const requestData = {
+      ...formData,
+      submitterUsername: user.AD_Username,
```

Comment:
Why are we changing submitter username here?  If this report was submitted by an admin user, the submitter username should stay as the admin user's name, so should never be changed.  If we want to record who marked it as picked up, that should be done through a report action, which is a separate API route (like what Ben has been working on for creating, deleting, and editing actions).

Reply:
Im just doing the same thing as Report Submit, because i cant just pass the report data, it wants me to specify the remaining fields, if that makes sense.

Reply:
Oh gotcha, yeah guess we didn't catch that at that point, but we definitely should change that for both.  I think you should be able to add those fields when you first fetch the form data from the API, and just don't have them appear in a textField, so they already exist in the type.

Reply:
It would take a bit more thinking through it to store that data. I would honestly not want that to be handled in this pull request, could be a bug fix i think, but for now, I'm thinking keeping it the way it is and tackling that seperatly.

Reply:
Just pushed changes that fix this, ended up being pretty simple.